HW04.Rmd

```diff
@@ -1,126 +1,273 @@
+    theme_bw()
+```
+
+(hint maybe looking at percentages would be good)
+```{r race_2}
+
+#add a percent column summarizing the info
+gun_death_percent<- gun_death_race3 %>%
+    group_by(race, intent) %>%
+    count() %>%
+    rename(death=n) %>%
+    group_by(race) %>%
+    mutate(total=sum(death), percent=death/total*100)
+
+#plot
+ggplot(gun_death_percent, aes(x=race, y=percent, fill=intent))+
+    geom_bar(stat="identity", position="stack", alpha=0.9)+
+    labs(title="Proportions of Reported Intentions of Gun Deaths Among Three Populations", x="Race", y="Percentage(%)", fill="Reported Intentions")+
+    theme_bw()
+```
+
+### Are police-involved gun deaths significantly different from other gun deaths? Assess the relationship between police involvement and age, police involvement and race, and the intersection of all three variables.
+
+```{r police}
+#data frame
+gun_death_police<- gun_death %>%
+    select(sex, age, police) 
+
+#make police variable factor
+gun_death_police$police<- factor(gun_death_police$police)
+
+#Are police-involved gun deaths significantly different from other gun deaths?
+gun_death_police %>%
+    group_by(police) %>%
+    count()
+#plot
+ggplot(gun_death_police, aes(x=police))+
+    geom_bar(fill="blue", alpha=0.7)+
+    labs(title="The Majority of Gun Deaths in 2012 Did Not Involve A Police", x="Police Involvement\n(No=0, Yes=1)", y="Number of deaths")+
+    theme_bw()
+```
+
+## Comparing the mean ages of gun death victims with or without police involvement, which is younger?
+
+- A t test comparing the means of age among gun deaths w/o police involvement shows that there is a statistically significant difference the age means. 
+- A boxplot below to visualize that the average age of the vicimts of police-involved gun deaths is younger. 
+
+```{r police_&_age}
+#make age variable numeric
+gun_death_police$age<- as.numeric(gun_death_police$age)
+
+#t test
+t.test(gun_death_police$age ~ gun_death_police$police)
+
+#plot
+ggplot(gun_death_police, aes(x=police, y=age, fill=police))+
+    geom_boxplot(alpha=0.9)+
+    labs(title="Age Differences of Gun Deaths w/o Police Involvement", x="Police Involvement\n(No=0, Yes=1)", y="Age")+
+    theme_bw()
+```
+
+## Are victims of police-involved gun death more likely to be male or female?
+
+- The visualization shows a greater proportion of police-involved victims are male compared to gun death that did not invovle a police. 
+- A Chi-square test shows statistical sigficance of this comparison.
+
+```{r police_&_sex}
+police_sex_percent<- gun_death_police %>%
+    group_by(sex, police) %>%
+    count() %>%
+    group_by(sex) %>%
+    mutate(total=sum(n), percent=n/total)
+
+ggplot(police_sex_percent, aes(x=police, y=percent, fill=sex))+
+    geom_bar(stat="identity", position="fill", alpha=0.7)+
+    labs(title="Police Involvement in Gun Deaths By Sex", x="Police Involvement\n(No=0, Yes=1)", y="Percent(%)")+
+    theme_bw()
```

Comment:
Actually, I just found out that I really like the colors of theme_bw(). I think I will use it in my future codes too!